cbmc_viewer/markup_link.py

```diff
@@ -40,10 +40,21 @@ def path_to_file(dst, src):
 ################################################################
 # Method to link into the source tree.
 # By default, links are from the root of the source tree to the source file.
+#
+# In what follows, the text being linked into the source tree may be
+# None.  For example, the source location for a global variable (a
+# static variable) will give a file name and a line number, but the
+# function name will be omitted (because there is no enclosing
+# function to name).  This may result in one of the following
+# functions being invoked with text set to None as the value of the
+# missing function name.
 
 def link_text_to_file(text, to_file, from_file=None, escape_text=True):
     """Link text to a file in the source tree."""
 
+    if not text:
+        return text
```

Comment:
Is it worth to add any logging info here?

Reply:
I think it would be noise in the debugging output.